CHANGES.md

```diff
@@ -120,8 +120,12 @@ Vernacular commands
 
 - `Declare Instance` now requires an instance name.
 
-- Option `Refine Instance Mode` has been turned off by default, meaning that
-  `Instance` no longer opens a proof when a body is provided.
+- Option `Refine Instance Mode` has been turned off and removed, meaning that
+  `Instance` no longer opens a proof when a body is provided. This is a
+  breaking change. We recommend to port impacted developments by first
+  turning the option off in 8.9 (using `Unset Refine Instance Mode`). Note
```

Comment:
I think it may be as easy to just run in master/8.10 and replace failing `Instance foo : Thing := bla. Proof. etc.` by `Instance foo : Thing. Proof. refine bla. etc.`

One may think that there's also the case where the type is gotten from the body, but 
~~~coq
Existing Class True.

Instance x : _ := I.
~~~
gives anomaly grounding non evar-free term so I guess there's no such case. (and `Instance x := I.` doesn't parse)